src/include/btmem.h

```diff
@@ -1575,8 +1589,22 @@ struct __wt_update {
      */
     wt_shared volatile uint64_t txnid; /* transaction ID */
 
-    wt_timestamp_t durable_ts; /* timestamps */
-    wt_timestamp_t start_ts;
+    union {
+        struct {
+            wt_timestamp_t durable_ts; /* timestamps */
+            wt_timestamp_t start_ts;
+        } commit;
+
+        struct {
+            wt_timestamp_t rollback_ts; /* rollback timestamp */
+            uint64_t saved_txnid;       /* transaction id before rollback */
+        } prepare_rollback;
+    } u;
+
+#define upd_durable_ts u.commit.durable_ts
```

Comment:
Do we really need the prefix `upd`? same for the `pg_del`? The structure itself is indicating from where it is coming.

Also, we need to #undefine them first and then #define.

Reply:
If we just use durbable_ts and start_ts, compilation will fail as start_ts is used in many other places. Adding upd and page_del is to ensure it is unique.